model/labels/labels.go

```diff
@@ -617,6 +617,12 @@ func (b *ScratchBuilder) Add(name, value string) {
 	b.add = append(b.add, Label{Name: name, Value: value})
 }
 
+// Add a name/value pair, using []byte instead of string.
+// Intended for '-tags stringlabels'; this version purely so the old way still compiles.
```

Comment:
```suggestion
// Intended for '-tags stringlabels'; this version has the same same (safe) semantics as Add.
```

Hm - this is bit weird - unsafe is sometimes safe sometimes unsafe. Why not making this use yolo string as well?

Reply:
Like it feels we need to ensure liveness of those bytes on caller side of this no matter what tag (labels implementation) is used.

Reply:
We have to copy some time. `stringlabels` makes one block of memory with all the values in, so that's a copy.  Previously all versions copied in `UvarintStr()`.

I don't think it's a great idea to make the documentation say it's safe.

Reply:
I pushed a commit which I hope makes the description clearer.  Let me know what you think.